service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/generated/DigitalTwins.java

```diff
@@ -0,0 +1,370 @@
+     */
+    Observable<ServiceResponseWithHeaders<Void, DigitalTwinUpdateDigitalTwinHeaders>> updateDigitalTwinWithServiceResponseAsync(String id, List<Object> digitalTwinPatch);
+    /**
+     * Updates a digital twin.
+     *
+     * @param id Digital Twin ID.
+     * @param digitalTwinPatch json-patch contents to update.
+     * @param ifMatch the String value
+     * @throws IllegalArgumentException thrown if parameters fail the validation
+     * @throws RestException thrown if the request is rejected by server
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent
+     */
+    void updateDigitalTwin(String id, List<Object> digitalTwinPatch, String ifMatch);
+
+    /**
+     * Updates a digital twin.
+     *
+     * @param id Digital Twin ID.
+     * @param digitalTwinPatch json-patch contents to update.
+     * @param ifMatch the String value
+     * @param serviceCallback the async ServiceCallback to handle successful and failed responses.
+     * @throws IllegalArgumentException thrown if parameters fail the validation
+     * @return the {@link ServiceFuture} object
+     */
+    ServiceFuture<Void> updateDigitalTwinAsync(String id, List<Object> digitalTwinPatch, String ifMatch, final ServiceCallback<Void> serviceCallback);
+
+    /**
+     * Updates a digital twin.
+     *
+     * @param id Digital Twin ID.
+     * @param digitalTwinPatch json-patch contents to update.
+     * @param ifMatch the String value
+     * @throws IllegalArgumentException thrown if parameters fail the validation
+     * @return the {@link ServiceResponseWithHeaders} object if successful.
+     */
+    Observable<Void> updateDigitalTwinAsync(String id, List<Object> digitalTwinPatch, String ifMatch);
+
+    /**
+     * Updates a digital twin.
+     *
+     * @param id Digital Twin ID.
+     * @param digitalTwinPatch json-patch contents to update.
+     * @param ifMatch the String value
+     * @throws IllegalArgumentException thrown if parameters fail the validation
+     * @return the {@link ServiceResponseWithHeaders} object if successful.
+     */
+    Observable<ServiceResponseWithHeaders<Void, DigitalTwinUpdateDigitalTwinHeaders>> updateDigitalTwinWithServiceResponseAsync(String id, List<Object> digitalTwinPatch, String ifMatch);
+
+    /**
+     * Invoke a digital twin root level command.
+     * Invoke a digital twin root level command.
+     *
+     * @param id the String value
+     * @param commandName the String value
+     * @throws IllegalArgumentException thrown if parameters fail the validation
+     * @throws RestException thrown if the request is rejected by server
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent
+     * @return the Object object if successful.
+     */
+    Object invokeRootLevelCommand(String id, String commandName);
+
+    /**
+     * Invoke a digital twin root level command.
+     * Invoke a digital twin root level command.
```

Comment:
duplicate summaries on these

Reply:
this is generated code, I wonder why this happened.

Reply:
same thing on line 153

Reply:
and everywhere on this file.

Reply:
I'm guessing both summary and description have the same value?

Reply:
Even if I remove it now, when we generate again it may be there. Do we want to manually update these generated files every time?

Reply:
For now I have removed here and below.